lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -272,13 +294,68 @@ static LogicalResult verifyCellOp(CellOp cell) {
 //===----------------------------------------------------------------------===//
 static LogicalResult verifyAssignOp(AssignOp assign) {
   auto parent = assign->getParentOp();
-  if (!(isa<GroupOp>(parent) || isa<WiresOp>(parent)))
+  if (!isa<GroupOp, WiresOp>(parent))
```

Comment:
BTW, I see you are using ParentOneOf... could you apply that trait to AssignOp and just remove this verifier?